openadmet/models/tests/inference/test_inference.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+import pandas as pd
+
+import pytest
+
+from openadmet.models.inference.inference import predict
+
+def test_predict():
+    # Test the predict function with a sample input
+    input_path = "tests/test_data/test_input.csv"
```

Comment:
You need to import these as fixtures rather than hardcoding the path, see the other test files for how this is done. 

Something like:
```python
from openadmet.models.tests.datafiles import intake_cat, test_csv
```